internal/stack/boot.go

```diff
@@ -43,6 +43,13 @@ func BootUp(options Options) error {
 		}
 	}
 
+	fmt.Println("Packages from the following directories will be loaded into the package-registry:")
+	fmt.Println("- built-in packages (package-storage:snapshot Docker image)")
+
+	if found {
+		fmt.Printf("- %s (copy of: %s)\n", stackPackagesDir, buildPackagesPath)
```

Comment:
I think we can just talk about `buildPackagesPath` here. Not sure if we should talk about `stackPackagePath` as it exposes `elastic-package` internals?

Reply:
Maybe you're right, I will leave only the `buildPackagesPath`

Reply:
Fixed.